access-control-app/docs/api.md

```diff
@@ -45,17 +45,13 @@ This defines common headers on the Access Control API.
 
 Content-Type is a standard HTTP header that specifies the content type of the body of the request. Access Control only supports `application/json` as the representation type for data.
 
-#### <a name="echo-token-header"></a> Echo-Token Header
-
-All Access Control API operations require specifying a token obtained from URS or ECHO. The token should be specified using the `Echo-Token` header.
-
 #### <a name="authorization-header"></a> Authorization Header
 
-The token can alternatively be specified using the `Authorization: Bearer` header, and by specifying a Bearer token.
+All Access Control API operations require specifying a token obtained from [Earthdata Login (EDL)](https://urs.earthdata.nasa.gov). The token should be specified using the `Authorization: Bearer` header followed by the EDL bearer token. For more information on obtaining an EDL bearer token, please reference the documentation [here](https://urs.earthdata.nasa.gov/documentation/for_users/user_token).
 
 #### <a name="cmr-revision-id-header"></a> Cmr-Revision-Id Header
 
-The revision id header allows specifying the revision id to use when saving the concept.  It is optional for all acl update.  The update will be rejected when 1. the revision id specified is not an integer. 2. the revision id specified <= current revision id of the acl - a HTTP Status code of 409 will be returned indicating a conflict.
+The revision id header allows specifying the revision id to use when saving the concept. It is optional for all acl update. The update will be rejected when 1. the revision id specified is not an integer. 2. the revision id specified <= current revision id of the acl - a HTTP Status code of 409 will be returned indicating a conflict.
```

Comment:
I think the 1. and 2. were meant to be a markdown list which somehow got turned into text.